submissions/Eugene-CG/DOM/index.js

```diff
@@ -61,7 +61,7 @@ const fillMenuList = (event) => {
     menuList.appendChild(li);
 
     const a = document.createElement("A");
-    a.setAttribute("href", "#");
+    a.setAttribute("href", "");
```

Comment:
For future works better use `#` instead of the empty `href` attribute.

Reply:
Got it!Thanks :3